web-components/src/components/inputs/TextField.tsx

```diff
@@ -130,6 +131,17 @@ const useStyles = makeStyles<Theme, StyleProps>((theme: Theme) => ({
       },
     },
   }),
+  default: {
+    '&:first-of-type': {
+      marginTop: 0,
+    },
+    [theme.breakpoints.up('sm')]: {
+      marginTop: theme.typography.pxToRem(40),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginTop: theme.typography.pxToRem(33),
+    },
+  },
```

Comment:
both `TextField` (hence all field components that use it) and `ControlledTextField` now use this default style, unless you specify `defaultStyle` prop as false
this avoids a lot of repeated styles
cc/ @mhagel @victordelval that will be useful for your issues https://github.com/regen-network/regen-registry/issues/816 and https://github.com/regen-network/regen-registry/issues/809